lib/subfont.js

```diff
@@ -136,6 +136,16 @@ module.exports = async function subfont(
 
   await assetGraph.checkIncompatibleTypes();
 
+  for (const relation of assetGraph
+    .findRelations({ type: 'HttpRedirect' })
+    .sort((a, b) => a.id - b.id)) {
+    if (relation.from.isInitial) {
+      console.warn(`${relation.from.url} redirected to ${relation.to.url}`);
```

Comment:
This should probably be `assetGraph.emit` instead. I guess info level is appropriate?

Reply:
Good point, fixed in bf15531